### PR TITLE
fix(discover): misc android ripple bounds

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Adjust the ripple for three components on Android:
- Learn more Link in Earn Pool Info Screen
- Reward Card on Discover
- Active Pools Card on Discover

#### Learn More

| Learn More Before (Android) | Learn More After (Android) |
| ----- | ----- |
|  |  | 

#### Reward Card
| Reward Card Before (Android) | Reward Card After (Android) |
| ----- | ----- |
|  |  |

#### Active Pools Card
| Active Pools Before (Android) | Active Pools After (Android) | 
| ----- | ----- |
| Screenshot 2024-10-21 at 12 04 00 PM |  |

### Test plan

- [x] Tested locally on Android
- [x] Tested locally on iOS

### Related issues

N/A

### Backwards compatibility

Yes

### Network scalability

N/A

### CODE REVIEW

File: src/earn/EarnActivePools.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export default function EarnActivePools() {
 
   return (
     <Touchable
+      shouldRenderRippleAbove
```
Comment (satish-ravi) [inline src/earn/EarnActivePools.tsx:55]: for my understanding, we don't seem to use this for the other Touchable, why is this required here?

File: src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx [hunk 2]
```diff
@@ -546,7 +546,7 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'center',
-    marginBottom: Spacing.Thick24,
+    padding: variables.contentPadding,
```
Comment (satish-ravi) [inline src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx:549]: not a fan of `variables.contentPadding`, as we have different padding for different elements throughout the app. Can we not use this? or define a consistent way of using this (e.g., have a clear definition of when to use this vs directly using Spacing)? At this point this just seems a proxy for a specific Spacing value.

File: src/earn/EarnActivePools.tsx (hunk 1 above)
Comment (MuckT) [reply src/earn/EarnActivePools.tsx:55]: We use it in [`src/points/PointsDiscoverCard.tsx`](https://github.com/valora-inc/wallet/blob/main/src/points/PointsDiscoverCard.tsx#L47) and it seems like the simplest way to get the ripple behavior we want.

File: src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx (hunk 2 above)
Comment (MuckT) [reply src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx]: Switch to Spacing in 26d57378f43b80076c56bbc0aaa75059759ef423.

File: src/earn/EarnActivePools.tsx (hunk 1 above)
Comment (satish-ravi) [reply src/earn/EarnActivePools.tsx:55]: ok, why is it not needed in DappFeaturedActions, but needed here?

**Review by satish-ravi — APPROVED** [review]

File: src/earn/EarnActivePools.tsx (hunk 1 above)
Comment (MuckT) [reply src/earn/EarnActivePools.tsx:55]: Linking some previous discussion around `shouldRenderRippleAbove` for posterity: https://github.com/valora-inc/wallet/pull/5409#discussion_r1604612862.
